src/components/common/TransactionalFolioNumber.vue

```diff
@@ -1,12 +1,15 @@
 <template>
-  <v-card flat id="transactional-folio-number-container" :class="{ 'invalid-section': !isValid }">
+  <v-card flat id="transactional-folio-number" class="py-8 px-6"
+    :class="{ 'invalid-section': !isValid }"
+  >
     <v-row no-gutters>
-      <v-col cols="3">
-        <label :class="{ 'error-text': !isValid }">
-          <strong>Folio or Reference<br>Number</strong>
+      <v-col cols="12" sm="3" class="pr-4 pb-4">
+        <label class="title-label" :class="{ 'error-text': !isValid }">
+          <strong>Folio or Reference Number</strong>
```

Comment:
No need for line break (br) -- we can let the column width (and gutter) handle it.

Note that some label columns have bottom padding (whitespace to data when stacked) and others don't. Example:

![image](https://user-images.githubusercontent.com/10002889/158040313-4198fa5b-f0d6-4f43-bfc5-4843c89a5a4e.png)

Reply:
Another example:

![image](https://user-images.githubusercontent.com/10002889/158040356-f4b289ab-24e9-4f62-8135-f7105d39c89b.png)

Reply:
I did what looked right for the data being displayed.